src/prose/prose.module.css

```diff
@@ -0,0 +1,292 @@
+:root {
+    --reactist-prose-code-fill: rgb(247, 250, 251);
+    --reactist-prose-code-tint: var(--reactist-content-primary);
+    --reactist-prose-quote-tint: #4a6368;
+    --reactist-prose-link-tint: var(--reactist-actionable-tertiary-idle-tint);
+}
+
+/* Internals for spacing. These are not considered public API. */
+:root {
+    --reactist-prose-content-font-size: 15px;
+    --reactist-prose-em-multiplier: 1.5;
+    --reactist-prose-space-1: calc(
+        var(--reactist-prose-em-multiplier) * var(--reactist-prose-content-font-size) / 3
+    );
+    --reactist-prose-space-2: calc(
+        var(--reactist-prose-em-multiplier) * var(--reactist-prose-content-font-size) * 2 / 3
+    );
+    --reactist-prose-space-3: calc(
+        var(--reactist-prose-em-multiplier) * var(--reactist-prose-content-font-size) * 1
+    );
+    --reactist-prose-space-4: calc(
+        var(--reactist-prose-em-multiplier) * var(--reactist-prose-content-font-size) * 4 / 3
+    );
+    --reactist-prose-space-6: calc(
+        var(--reactist-prose-em-multiplier) * var(--reactist-prose-content-font-size) * 5 / 3
+    );
+}
+
+.prose {
+    font-size: var(--reactist-prose-content-font-size);
+    color: var(--reactist-content-primary);
+    line-height: 1.65;
+    overflow-wrap: break-word;
+    word-wrap: break-word;
+    word-break: break-word;
+}
+
+.prose,
+.prose *,
+.prose *::before,
+.prose *::after {
+    box-sizing: border-box;
+}
+
+.prose > :first-child,
+.prose :global(.ProseMirror > *:first-child) {
+    margin-top: 0;
+}
+
+.prose > :last-child,
+.prose :global(.ProseMirror > *:last-child) {
+    margin-bottom: 0;
+}
+
+.prose :global(br.ProseMirror-trailingBreak:only-child) {
+    font-size: inherit;
+}
```

Comment:
These hardcoded references to a ProseMirror class name is something I want to figure out next week, maybe with the help of Ricardo, how we can make this work seamlessly without Reactist having to have knowledge about it.

Reply:
Do you want to discuss this here or over Twist? Perhaps in a sync fashion? Either way is fine for me :)

Reply:
A sync video call should be helpful and allow us to move faster. I'll ping you on Twist to coordinate.